pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -1,8 +1,6 @@
 { stdenv, fetchgit, fetchFromGitHub, cmake, pkgconfig, git, python3,
   python3Packages, glslang, spirv-tools, x11, libxcb, wayland }:
 
-assert stdenv.system == "x86_64-linux";
```

Comment:
`meta.platforms` list is probably the best place. Only three system values are commonly used, I think, so the rest seems not too much significant anyway.